docs/docs/config-colors.md

```diff
@@ -51,5 +51,15 @@ To change *only* the background color, just omit the first color from the above
 "prefix": "<,#FFFFFF>┏[</>",
 ```
 
+## Palette reference
+
+If your theme has a defined [Palette][palette], you can use the Palette reference in places where the
+__Standard color__ is expected.
+
+Palette references have 2 possible formats: `p:<palette key>` and `palette:<palette key>`.
```

Comment:
let's remove `palette:<palette key>`, not necessary.

Reply:
Removed.